go/vt/vtgate/planbuilder/select.go

```diff
@@ -205,3 +209,21 @@ func (pb *primitiveBuilder) pushSelectRoutes(selectExprs sqlparser.SelectExprs)
 	}
 	return resultColumns, nil
 }
+
+// queryTimeout returns true/value if DirectiveQueryTimeout is set.
+func queryTimeout(d sqlparser.CommentDirectives) (bool, int) {
```

Comment:
Should we make this simpler and just return 0 if it's not set, since that's the default nil value anyway?